ui/widgets/nuxeo-operation-button.js

```diff
@@ -235,6 +267,112 @@ import '../actions/nuxeo-action-button-styles.js';
     _computeTooltip(tooltip, label) {
       return tooltip || label;
     }
+
+    _isPageProviderDisplayBehavior(input) {
+      return (
+        input &&
+        input.behaviors &&
+        Nuxeo.PageProviderDisplayBehavior &&
+        Nuxeo.PageProviderDisplayBehavior.every((p) => input.behaviors.includes(p))
+      );
+    }
+
+    _isSelectAllActive(input) {
+      return this._isPageProviderDisplayBehavior(input) && input.selectAllActive;
+    }
```

Comment:
We share this code with nuxeo-operation. I think we should have a follow up ticket to refactor it.

Reply:
It is still under review the need to have it on both elements. If we really need them on both files, I'll open a ticket to refactor it.

Reply:
We no longer share the code between the `nuxeo-operation` and `nuxeo-operation-button` because in `nuxeo-operation` we don't have the behavior available.